frontstage/views/secure_messaging/message_get.py

```diff
@@ -48,8 +49,13 @@ def view_conversation(session, thread_id):
             return redirect(url_for('secure_message_bp.view_conversation_list'))
 
     unread_message_count = {'unread_message_count': get_message_count_from_api(session)}
-    survey_name = get_survey(app.config['SURVEY_URL'], app.config['BASIC_AUTH'],
-                             refined_conversation[-1]['survey_id']).get('longName')
+    survey_name = None
```

Comment:
would be good to have a unit test which checks this is rendered as "No survey" in this scenario.

Reply:
Done, thanks.

Reply:
Done, thanks